addons/stock_account/models/product.py

```diff
@@ -200,7 +195,8 @@ def _get_candidates_move(self):
             ('location_dest_id.usage', 'in', ('transit', 'internal')),
             ('location_id.usage', 'not in', ('transit', 'internal')),
             ('remaining_qty', '>', 0),
-            ('state', '=', 'done')
+            ('state', '=', 'done'), 
+            ('company_id', '=', self.company_id.id)
         ], order='date, id') #TODO: case where 
         return candidates
 
```

Comment:
on `_compute_average_price` is useful to have a meaningful value of `average_price` != 0 
because if for some reason the product get depleted and you want to adjust inventory, and 
it turns out that there was products available, you then end up with adjusting a product
with valuation zero, 

https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_card/model/stock_card.py#L460-L463

We have witnessed this scenario with our customers.